fedimint-server/src/config/api.rs

```diff
@@ -1128,8 +1130,8 @@ mod tests {
 
         apis.push(api);
 
-        for i in 1..4 {
-            let port = base_port + (i * 2);
+        for i in 1..PEER_NUM {
+            let port = base_port + (i * PORTS_PER_PEER);
```

Comment:
Why not just call `port_alloc(PORTS_PER_PEER)` every time instead of doing these manual calculations?

Reply:
:brain: